lib/assessment.js

```diff
@@ -8,15 +8,45 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
 
+// iterate over the array, use ternary operator to compare currentValue
+// to previousValue and assign the min to whichever satisfies
+// the condition
+var min = function min(previousValue, currentValue, index, array) {
+  // set the intial result
+  var result = previousValue;
+  for (var i = 0; i < array.length; i++) {
```

Comment:
This function is meant to be called by a function that handles iterating over the array for you. That's where you get `previousValue`, `currentValue`, and `index` from. That you're iterating over the array in here means you're iterating over the array while iterating over the array.